phishing_protection/google/cloud/phishingprotection_v1beta1/types.py

```diff
@@ -27,7 +27,7 @@
 
 names = []
 
-for module in _shared_modules:
+for module in _shared_modules:  # pragma: NO COVER
```

Comment:
Hmm, how are we going to keep this fix from being removed in the next autogen?

Reply:
I didn't think about this at the time :woman_facepalming:. I've raised an issue asking for the loop to be omitted when there are no shared_modules https://github.com/googleapis/gapic-generator/issues/2770